src/api/expenses/expenses.router.js

```diff
@@ -0,0 +1,18 @@
+const { Router } = require('express');
+const { expensesController } = require('./expenses.controller');
+
+const expensesRouter = Router();
+
+expensesRouter.get('/', expensesController.getAll);
+
+expensesRouter.post('/', expensesController.create);
```

Comment:
```suggestion
expensesRouter.post('/', your__middleware, expensesController.create);
```
Consider adding the middleware and returning the 400 error here